templates/types/extractor/fastapi/app/api/routers/extractor.py

```diff
@@ -12,4 +12,4 @@
 
 @r.post("/query")
 async def query_request(data: RequestData):
-    return ExtractorService.extract(data)
\ No newline at end of file
+    return ExtractorService.extract(query=data.query, json_schema=None)
```

Comment:
can already pass the schema (if `None` then use the default schema)
```suggestion
    return ExtractorService.extract(query=data.query, json_schema=data.json_schema)
```